src/base/bittorrent/torrent.h

```diff
@@ -193,7 +193,8 @@ namespace BitTorrent
         virtual qreal ratioLimit() const = 0;
         virtual int seedingTimeLimit() const = 0;
 
-        virtual QStringList absoluteFilePaths() const = 0;
+        virtual QVector<QString> files() const = 0;
+        virtual QVector<QString> absoluteFilePaths() const = 0;
```

Comment:
what is the difference of `files()` and `absoluteFilePaths()`?

Reply:
`absoluteFilePaths()` is one more (IMO, redundant) helper that just returns all file paths in absolute form (i.e. "save path" + "relative file path").